packages/material-ui/src/MobileStepper/MobileStepper.js

```diff
@@ -91,29 +159,28 @@ const MobileStepper = React.forwardRef(function MobileStepper(props, ref) {
       )}
 
       {variant === 'dots' && (
-        <div className={classes.dots}>
+        <MobileStepperDotContainer styleProps={styleProps}>
           {[...new Array(steps)].map((_, index) => (
-            <div
+            <MobileStepperDot
               key={index}
-              className={clsx(classes.dot, {
-                [classes.dotActive]: index === activeStep,
-              })}
+              className={clsx(classes.dot)}
+              styleProps={{ ...styleProps, dotActive: index === activeStep }}
             />
           ))}
-        </div>
+        </MobileStepperDotContainer>
       )}
 
       {variant === 'progress' && (
-        <LinearProgress
-          className={classes.progress}
+        <MobileStepperLinearProgress
+          styleProps={styleProps}
```

Comment:
```suggestion
          styleProps={styleProps}
          className={classes.linearProgress}
```

On this note, I noticed that you renamed the `dots` to `dotsContainer` and `progress` to `linearProgress`. Was this intentional? If we rename this we should state it as breaking change. I'd say let's not change it unless really necessary. (also if we bring the old names, we should change the names of the slots components)

Reply:
Sorry I did not see 'slots' / classes for these. I will make the changes back. Was *not* trying to intro a breaking change